app/uk/gov/hmrc/gform/translation/TextExtractor.scala

```diff
@@ -638,38 +631,181 @@ object TextExtractor {
+
+      def pathToSectionNumber(path: String) = {
+        val index = getPathIndex(path)
+        val sectionType = index.flatMap(pathIndexToSectionType.get)
+        val pageNumber = getPageNumberFromPath(path)
+
+        def getClassic(normalIndex: Int, sectionType: SectionType) = {
+          val index = TemplateSectionIndex(normalIndex)
+          sectionType match {
+            case ClassicNormal    => Some(Classic.NormalPage(index))
+            case ClassicRepeating => Some(Classic.RepeatedPage(index, 0))
+            case ClassicAddToList =>
+              pageNumber match {
+                case Some(pageNumber)                             => Some(Classic.AddToListPage.Page(index, 1, pageNumber))
+                case None if path.contains(".defaultPage")        => Some(Classic.AddToListPage.DefaultPage(index))
+                case None if path.contains(".cyaPage")            => Some(Classic.AddToListPage.CyaPage(index, 1))
+                case None if path.contains(".addAnotherQuestion") => Some(Classic.AddToListPage.RepeaterPage(index, 1))
+                case None if path.contains(".declarationSection") =>
+                  Some(Classic.AddToListPage.DeclarationPage(index, 1))
+                case _ => None
+              }
+          }
+        }
+
+        index -> sectionType match {
+          case (Some(PathIndex(normalIndex, None, None)), Some(sectionType)) =>
+            getClassic(normalIndex, sectionType)
+          case (Some(PathIndex(normalIndex, Some(taskIndex), Some(taskSectionIndex))), Some(sectionType)) =>
+            getClassic(taskSectionIndex, sectionType).map { classic =>
+              val coordinates = Coordinates(TaskSectionNumber(normalIndex), TaskNumber(taskIndex))
+              TaskList(coordinates, classic)
+            }
+          case (Some(PathIndex(normalIndex, Some(taskIndex), taskSectionIndex)), None)
+              if !containAnyOf(path, summarySectionPaths) =>
+            Some(Classic.AddToListPage.Page(TemplateSectionIndex(normalIndex), 1, 1))
+          case (Some(PathIndex(normalIndex, None, None)), None) if !containAnyOf(path, summarySectionPaths) =>
+            Some(Classic.NormalPage(TemplateSectionIndex(normalIndex)))
+          case _ => None
+        }
+      }
+
+      def sectionText(paths: Seq[String]) =
+        paths
+          .map { path =>
+            path -> pathToSectionNumber(path)
+          }
+          .collect {
+            case (path, Some(sectionNumber))                              => "screenshot-" + sectionNumber.value + ".png"
+            case (path, None) if path.contains(".formName")               => "any"
+            case (path, None) if path.contains(".submitSection")          => "submit section"
+            case (path, None) if path.contains(".acknowledgementSection") => "acknowledgement section"
+            case (path, None) if containAnyOf(path, summarySectionPaths)  => "summary section"
```

Comment:
`containAnyOf(path, summarySectionPaths)` is not covering all fields which appears on summary section.

For example assumoing 0 is atl
```
.sections[0].shortName
```
shows on summary page.